tests/cnf/ran-deployment/deploymenttypes/internal/gitdetails/gitdetails.go

```diff
@@ -0,0 +1,24 @@
+package gitdetails
+
+import (
+	"fmt"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/argocd"
+	"github.com/openshift-kni/eco-goinfra/pkg/schemes/argocd/argocdtypes/v1alpha1"
```

Comment:
```suggestion
	argocdtypes "github.com/openshift-kni/eco-goinfra/pkg/schemes/argocd/argocdtypes/v1alpha1"
```
really I should've named the package better but too late now. It helps readability though to not use just the version as the package name, however

Reply:
using argocdtypes instead of v1alpha1